### PR TITLE
Add Support for XinFin ( XDC ) Network.

### PR DESCRIPTION
**Pull request by BlocksScanIO**
XinFin XDC.Network is an enterprise-ready hybrid Blockchain technology company optimised for international trade and finance. The XinFin XDC.network is powered by the native coin called XDC. The XDC protocol is architected to support smart contracts, 2000TPS, 2seconds transaction time, KYC to Masternodes (Validator Nodes). The XDC.Network uses XinFin Delegated Proof of Stake (XDPoS), with the intending to create a ‘highly-scalable, secure, permission, and commercial grade’ blockchain network.

Github:- https://github.com/XinFinOrg

CMC :- https://coinmarketcap.com/currencies/xinfin-network/

Docker FullNode:- https://github.com/xinfinorg/XinFin-Node

Technical Community Support available at:

Telegram Community: https://t.me/XinFinDevelopers

Slack Community: https://xinfin-public.slack.com/messages/CELR2M831/

Slack Invitation Link: https://launchpass.com/xinfin-public

Technical help Resource: http://howto.xinfin.org/

### CODE REVIEW

**Comment by BlocksScanIO** [comment]
Hello @FrederikBolding ,

Please guide us with further step to get XDC.Network listed in MyCrypto Wallet.

**Comment by FrederikBolding** [comment]
Hi! Thanks for opening the PR. I haven't had the time to look it through yet, but will try to get that done soon. Then there might be a few questions or changes requested. 

Thanks for your patience.

**Comment by BlocksScanIO** [comment]
Hello @FrederikBolding, 

Just the formal followup on when can we see the XinFin XDC.Network on MyCrypto.

**Comment by FrederikBolding** [comment]
> Hello @FrederikBolding,
> 
> Just the formal followup on when can we see the XinFin XDC.Network on MyCrypto.

Sorry for the delay - we've been very busy lately. Will try to get to this during this week!

File: jest_config/__fixtures__/assets.ts [hunk 1]
```diff
@@ -182,6 +182,15 @@ export const fAssets = [
     networkId: 'SmartChain',
     isCustom: false
   },
+  {
```
Comment (FrederikBolding) [inline jest_config/__fixtures__/assets.ts:185]: There is no need to add this. Please remove 👍

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

**Comment by FrederikBolding** [comment]
Also, if possible can you rebase your PR?

**Comment by BlocksScanIO** [comment]
Changes added and updated branch. :)

**Comment by BlocksScanIO** [comment]
Rebase done...!!!

**Comment by BlocksScanIO** [comment]
@FrederikBolding changes has been added.

**Review by FrederikBolding — APPROVED** [review]

**Comment by FrederikBolding** [comment]
@BlocksScanIO Merged! The network will be included in our next release, probably some time this week!

**Comment by bowensanders** [comment]
@BlocksScanIO Would it be possible, since we do not have any tokens for your network, for you to check a build to make sure your network is functioning properly before we push it live?
[test link](https://rc.app.MyCrypto.com)
Just add your account and perhaps do a send (to yourself or otherwise)?
Thanks!

**Review by popeda97 — COMMENTED** [review]
blocksscan:master

**Comment by ChainGamer** [comment]
> @BlocksScanIO Merged! The network will be included in our next release, probably some time this week!

Hi @FrederikBolding Has this been publicly released?

**Comment by BlocksScanIO** [comment]
image


@ChainGamer Yes its live 👍

**Pull request merged**